cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,140 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 6.0.0 <http://docs.cask.co/cdap/6.0.0/index.html>`__
+=============================================================
+
+New Features
+------------
+
+- :cask-issue:`CDAP-14330` - Added Google Cloud Storage copy and move action plugins
+
+- :cask-issue:`CDAP-14533` - New pipeline list user interface.
+
+- :cask-issue:`CDAP-14613` - Added minimap to pipeline canvas.
+
+- :cask-issue:`CDAP-14645` - Added support for running CDAP system services in Kubernetes environment.
+
+- :cask-issue:`CDAP-14657` - Add the ability to copy and paste a node in pipeline studio.
```

Comment:
Let's keep it consistent and always use "Added"
So, change "Add" to "Added" here and wherever else needed.